x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -288,7 +306,10 @@ const ActionsConnectorsList: React.FunctionComponent = () => {
             <DeleteOperation
               canDelete={canDelete}
               item={item}
-              onDelete={() => setConnectorsToDelete([item.id])}
+              onDelete={() => {
+                setConnectorsToDelete([item.id]);
+                setDeleteConnectorWarning([item.id]);
```

Comment:
What do you think of creating an onDelete function that makes both of these calls? Then you can also wrap it with a [useCallback](https://reactjs.org/docs/hooks-reference.html#usecallback). Could use this below in the bulkDelete onClick as well

```
const onDelete = useCallback((items) => {
  const itemIds = items.map((item: any) => item.id);
  setConnectorsToDelete(items);
  setDeleteConnectorWarning(items);
}, [<dependencies>]);
```

Reply:
Resolved in this [commit](https://github.com/elastic/kibana/pull/137181/commits/dac37a909ba1e0cb375cd2635557acd48219c868)